README.md

```diff
@@ -2,9 +2,16 @@
 
 Ansible based project to configure a multi-node tsuru cluster.
 
-### Requirement:
-* ansible 1.4 or higher.
-* librarian-ansible 1.0.6 or higher
+### Requirements:
+
+* Python things (you may wish to use [virtualenv](https://virtualenv.pypa.io/en/latest/)):
+```
+pip install -r requirements.txt
```

Comment:
If you use `-Ur` it'll also make sure we [upgrade to the latest specified package versions](https://pip.pypa.io/en/latest/reference/pip_install.html#cmdoption-U)